test/jasmine/tests/click_test.js

```diff
@@ -47,7 +47,7 @@ describe('Test click interactions:', function() {
             setTimeout(function() {
                 click(x, y);
                 setTimeout(function() { resolve(); }, DBLCLICKDELAY / 2);
-            }, DBLCLICKDELAY / 4);
+            }, DBLCLICKDELAY / 2);
```

Comment:
#567 is not fixed... still some intermittent failures around there. I'm not going to tweak it any more in this PR, but I think you're on to something with judicious event listening.